redis/src/test/scala/zio/redis/StringsSpec.scala

```diff
@@ -1435,6 +1437,92 @@ trait StringsSpec extends BaseSpec {
             len <- strLen(key).either
           } yield assert(len)(isLeft(isSubtype[WrongType](anything)))
         }
+      ),
+      suite("getEx")(
+        testM("value exists after removing ttl") {
+          for {
+            key    <- uuid
+            value  <- uuid
+            _      <- pSetEx(key, 1000.millis, value)
+            exists <- getEx(key, true)
+            _      <- ZIO.sleep(1010.millis)
```

Comment:
This is sleeping for a little longer than a second. I would reduce that by at least 10x to not slow down the tests too much. WDTY?

Reply:
Yes